phoenix-core/src/main/java/org/apache/phoenix/query/QueryServices.java

```diff
@@ -158,6 +158,7 @@ public interface QueryServices extends SQLCloseable {
     public static final String INDEX_FAILURE_HANDLING_REBUILD_INTERVAL_ATTRIB =
         "phoenix.index.failure.handling.rebuild.interval";
     public static final String INDEX_REBUILD_TASK_INITIAL_DELAY = "phoenix.index.rebuild.task.initial.delay";
+    public static final String INDEX_START_TRUNCATE_TASK_DELAY = "phoenix.index.start.truncate.task.delay";
```

Comment:
stats truncate is not related to index, so let's remove "index" from configs everywhere